File: WebTotalCommander/webtotalcommander.client/src/app/services/models/common/folder.getall.view-model.ts
import { FolderFileViewModel } from "./folder.file.view-model";
import { PaginationMetaDataView } from "./pagination.data";

export class FolderGetAllViewModel{
    folderFile:Array<FolderFileViewModel>=[];
    paginationMetaData:PaginationMetaDataView=new PaginationMetaDataView();
}

File: WebTotalCommander/WebTotalCommander.Service/ViewModels/Common/FolderGetAllViewModel.cs
using WebTotalCommander.FileAccess.Utils;

namespace WebTotalCommander.Service.ViewModels.Common;

public class FolderGetAllViewModel
{
    public List<FolderFileViewModel> FolderFile { get; set; } = new List<FolderFileViewModel>();
    public PaginationMetaData PaginationMetaData { get; set; } = new PaginationMetaData();
}
